week1/practice/ComposeQuadrant/app/src/main/java/com/example/composequadrant/MainActivity.kt

```diff
@@ -5,32 +5,63 @@ import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.*
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
-import com.example.composequadrant.ui.theme.ComposeQuadrantTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContent { }
+        setContent {
+            ComposeQuadrantApp()
+        }
     }
 }
 
 @Composable
-fun ComposeQuadrantApp() {
+fun ComposeQuadrantApp(m
+) {
```

Comment:
```suggestion
fun ComposeQuadrantApp(modifier: Modifier = Modifier) {
```
여기서 오류가 발생하고 있는 것 같습니다!
modifier 파라미터를 추가하고 Column의 인자로 modifier를 전달해주면 될 것 같아요 🙃